app/helpers/followers_helper.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module FollowersHelper
+
+  # render a follow or unfollow btn for specific user
+  def followers_btn(user)
+    if user_signed_in? && current_user.id != user.id
+      @@following = user.followed_by(current_user.id)
```

Comment:
@jshidalgokommit Why the double` @@`?

Reply:
Sorry, I thought it was a variable only for the class. I should have used only @